tox.ini

```diff
@@ -1,13 +1,13 @@
 [tox]
-envlist = py34, py35, py36, py36-dev, py37, pypy, lint
+envlist = py34, py35, py36, py37, py38-dev, pypy, lint
 skipsdist = True
 
 [testenv]
 usedevelop = True
 commands = py.test {posargs}
 deps =
   -rrequirements_test.txt
-  py{34,35,36,36-dev,37}: aiohttp
+  py{34,35,36,37,38-dev}: aiohttp
```

Comment:
Actually, we can move aiohttp into `requirements_test.txt` now.

Reply:
Right, will do - I previously noticed it was deprecated.